app/views/topics/new.html.erb

```diff
@@ -0,0 +1,22 @@
+<h2>Topics</h2>
+<p>Add a new topic</p>
+
+<div>
+  <%= form_for @topic do |f| %>
+    <%= f.label :name %>
+    <%= f.text_field :name, id: :name %>
```

Comment:
Why do you need the `id: :name`?

Reply:
I couldn't remember why so I removed it.... then ran my tests and got "Capybara:: ElementNotFound:  Unable to find visible field :name that is not disabled" - which made me remember, adding this ID fixes this problem... so I've added it back in... is there a better way of doing this?